### ISSUE TITLE
I can not select the scroll bar in datatable factor on Internet Explorer 11

### ISSUE THREAD

**Issue by ToshihiroIguchi** [rstudio/DT#556]
Hello,

I am using DT and I feel it is wonderful.
However, there is a problem that one can not solve, so I will ask you a question.

When choosing a factor with datatable, the scroll bar appears on the right side when the number of factors is large.
However, in Internet Explorer 11, when you click the scroll bar, the scroll bar disappears and it can not be selected.
I installed the latest version of DT from github but it did not solve it.
In Chrome and edge, I could scroll by clicking the scroll bar without problems.

Internet Explorer 11 Write the code that causes the problem below.

    library(DT)
    data <- data.frame(x = c(1:10), y = c("a","b","c","d","e","f","g","h","i","j"))
    datatable(data, filter = 'top')

If you know how to solve this problem with Internet Explorer 11, please let me know.

**Comment by ToshihiroIguchi** [comment]
Attach the image of the sidebar.
![sidebar](https://user-images.githubusercontent.com/20187797/41290544-2a4f8c32-6e88-11e8-8d4b-883acf191055.png)

**Comment by shrektan** [comment]
Thanks for reporting. I can confirm this bug. The selectize input seems not work well on IE11. Since `shiny::selectizeInput()` works good in IE11 (see the below code), it should be a problem in DT .

```r
library(shiny)
ui <- basicPage(DT::DTOutput("dt"))
server <- function(input, output, session) {
  output$dt <- DT::renderDT(
    data.frame(b = as.character(selectizeInput("a", NULL, choices = rep(letters, 10)))),
    escape = FALSE)
}
shiny::runApp(mget(c("ui", "server")))
```

**Comment by shrektan** [comment]
Should have been fixed in #557 , you may try `devtools::install_github("rstudio/DT#557")`.

**Comment by ToshihiroIguchi** [comment]
Hello,

I installed # 557 with devtools and confirmed that the scroll bar problem was solved in my IE 11 environment.

You have completely solved my problem.
Thank you. I appreciate it very much.
